doc/reference/autoinstall-reference.rst

```diff
@@ -711,7 +710,7 @@ install-server
 
 Whether to install OpenSSH server in the target system.
 
-authorized-keys
+:spellexception:`authorized-keys`
```

Comment:
wordlist please.  This is the necessary syntax for having the ssh functionality work.

Reply:
Ack